pisa/utils/stats.py

```diff
@@ -127,8 +127,6 @@ def chi2(actual_values, expected_values):
         #       still destroy a minimizer's hopes and dreams...
 
         # Replace 0's with small positive numbers to avoid inf in division
-        np.clip(actual_values, a_min=SMALL_POS, a_max=np.inf,
```

Comment:
why are all the clipping of values removed??

Reply:
aha, good point! in the old days we used the Neyman chi2 dividing by observed, but now in the Pearson chi2 this is not the case....so as you point out we can leave the actual values alone